src/main/java/it/arcade/hospital/testclass/TestClass.java

```diff
@@ -0,0 +1,21 @@
+package it.arcade.hospital.testclass;
+
+public class TestClass {
+
```

Comment:
Any reason to use TestClass instead of simply return String in Controller?

Reply:
I wanted return data in bracket json style. simple string just showed page with nothing. This is only one reason, maybe in future will more

Reply:
Are you sure that returning String with Restcontroller showed page with nothing? Below simple test
![string_code](https://user-images.githubusercontent.com/18702379/98916097-07a2c180-24cb-11eb-91e6-536132952c44.png)
Of course if you wanted to return simple Json, there is no need to reinvent the wheel by creating the separate object which contains only the String. Built in type Map does the job: 
![hashmap](https://user-images.githubusercontent.com/18702379/98916375-623c1d80-24cb-11eb-9677-057acbf63bfc.png)
So choose the way you like, and make modification because after creating the GUI there will be a need to clean the code, so the less files we have the better.

Reply:
My mistake, I wanted to say that simple string just show like on first screen :).
Second way is also nice but also here have few additional lines of code. I will check this and recode.

Reply:
With Java 11 you can use `return Map.of("test", "Test");`